### PR TITLE
workflows: use more Homebrew actions

### PR DESCRIPTION
**Pull request by dawidd6**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

https://github.com/Homebrew/brew/issues/7805

### CODE REVIEW

File: .github/workflows/publish-commit-bottles.yml [hunk 1]
```diff
@@ -63,47 +43,19 @@ jobs:
       - name: Push commits
         uses: Homebrew/actions/git-try-push@master
       - name: Post comment on failure
-        if: ${{!success()}}
-        uses: actions/github-script@0.9.0
+        if: ${{failure()}}
```
Comment (jonchang) [inline .github/workflows/publish-commit-bottles.yml:46]: We should keep this as `!success` since jobs can be cancelled and that should post a comment.

**Review by jonchang — DISMISSED** [review]
Looks good!

**Comment by jonchang** [comment]
Feel free to manually merge this when you're ready.

**Comment by glyng1** [comment]
I'm am having trouble with my laptop and S10+ confirmation.

Regards glyn

**Review by SMillerDev — COMMENTED** [review]
Lgtm

**Comment by glyng1** [comment]
Yes but having difficulty with my devices so 😞

**Review by MikeMcQuaid — COMMENTED** [review]
Great cleanup!

**Pull request merged**